llms/mlx_lm/tuner/trainer.py

```diff
@@ -227,6 +231,7 @@ def step(batch):
             batch_size=args.batch_size,
             max_seq_length=args.max_seq_length,
             train=True,
+            args=args,
```

Comment:
I thought about this a bit more and it should be easy enough to enclose the training args in any custom iterate batches function (that gets passed to train). So I don't think we really need to pass it through this way.